### ISSUE TITLE
studio(a11y): sign in button not reachable with keyboard only

### ISSUE THREAD

**Issue by donroyco** [deckgo/deckdeckgo#534]
## Current behaviour

When tabbing through the website on https://deckdeckgo.com/, it skips the 'Sign in' button and goes directly to 'Write presentation'.

![20191214-deckdeckgo-signin](https://user-images.githubusercontent.com/1763537/70848501-d19c2e00-1e72-11ea-8782-25c206603ca5.gif)
_Reproducing GIF of current behaviour._

Side note: when using MacOS built in VoiceOver, the Sign in link is inside the rotor menu.

## Expected behavior

It should not skip the 'Sign in' button.

**Comment by peterpeterparker** [comment]
@donroyco well spotted, thx for the issue!

**Comment by StefanNieuwenhuis** [comment]
Is this issue still relevant?

**Comment by peterpeterparker** [comment]
yes sure @StefanNieuwenhuis 

all improvements are welcome!

**Comment by StefanNieuwenhuis** [comment]
When I visit [https://deckdeckgo.com/](https://deckdeckgo.com/), I see a completely different site, without any sign-in button. Hence my question :)

**Comment by peterpeterparker** [comment]
@StefanNieuwenhuis that's brand new ;)

Summarized the welcome screen is:

not logged -> landing page (the one you see I guess)
logged -> the feed as before

There is now a landing page for not logged users. We thought it was nice to add one to explain a bit what's DeckDeckGo.

**Comment by peterpeterparker** [comment]
Solved in next version, thx for the issue @donroyco 👍
